src/components/Match/PlayerThumb/index.jsx

```diff
@@ -19,7 +19,7 @@ const StyledImg = styled.img`
 `;
 
 const PlayerThumb = ({
-  player_slot, hero_id, name, personaname, hideText,
+  player_slot, hero_id, name, personaname, hideText, // eslint-disable-line camelcase
```

Comment:
you can remove these eslint disables, they have been fixed in master (and this is causing a merge conflict)